python/lsst/ts/observatory/control/auxtel/atcalsys.py

```diff
@@ -196,11 +196,38 @@ async def prepare_for_flat(self, config_name: str) -> None:
             else utils.make_done_future()
         )
 
+        task_setup_electrometer = self.setup_electrometers(
+            mode=int(config_data["electrometer_mode"]),
+            range=float(config_data["electrometer_range"]),
+            integration_time=float(config_data["electrometer_integration_time"]),
+        )
+
         await asyncio.gather(
-            task_setup_monochromator,
-            task_setup_latiss,
+            task_setup_monochromator, task_setup_latiss, task_setup_electrometer
```

Comment:
This is a picky comment but do you mind adding a comma after `task_setup_electrometer`? This will cause `black` to format the code such that each task is in a separate line (like it was originally), thus reducing the signature change when we add new tasks.